pep-0008.txt

```diff
@@ -121,6 +121,51 @@ Optional::
       var_one, var_two,
       var_three, var_four)
 
+For dictionaries there is an extra rule. When placing a dictionary key on a new
```

Comment:
nitpicky 😅  there should be a comma
`For dictionaries, there is ... `

 `... on a new line, an additional ...`